prerequisites/install-functions/report_results.sh

```diff
@@ -69,10 +69,10 @@ report_results()
     if [[ -d "${compiler_install_root%/}/lib" || -d "${compiler_install_root%/}/lib64" ]]; then
       echo "# Prepend the compiler library paths to the ${LD_LIB_P_VAR} environment variable:" | tee -a setup.sh setup.csh
       compiler_lib_paths="${compiler_install_root%/}/lib64/:${compiler_install_root%/}/lib"
-      echo "if [[ -z \"\${!${LD_LIB_P_VAR}}\" ]]; then                                    " >> setup.sh
+      echo "if [[ -z \"\${${LD_LIB_P_VAR}}\" ]]; then                                    " >> setup.sh
```

Comment:
gah! Good catch. You are in fact correct that this is the correct logic to resolve this.